### PR TITLE
chore(monorepo): add github actions to automate release process

### PR DESCRIPTION
**Pull request by bardisg**
## Description of the change

Added github actions and prepared sonacloud config

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

[> Fix [#1]() ](https://www.notion.so/rudderstacks/CI-CD-pipeline-Flutter-bf41a6c20c284f41a3d72bc3e9d28705)

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [x]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [x] Changes have been reviewed by at least one other engineer
- [x] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by desusai7 — APPROVED** [review]

File: .github/workflows/deploy-pubdev.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          echo "CURRENT_VERSION_VALUE=$current_version" >> $GITHUB_ENV
+          echo "DATE=$(date)" >> $GITHUB_ENV
+
+      - name: Send message to Slack channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          PROJECT_NAME: 'Flutter SDK pub.dev packages'
+          NPM_PACKAGE_URL: 'https://pub.dev/packages?q=rudder`
```
Comment (itsdebs) [inline .github/workflows/deploy-pubdev.yml:66]: Shouldn't this be [https://pub.dev/packages/rudder_sdk_flutter](https://pub.dev/packages/rudder_sdk_flutter)

File: .github/workflows/draft-new-release.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+      - name: Create pull request into main
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: ${{ steps.create-release.outputs.branch_name }}
+          destination_branch: 'main'
+          github_token: ${{ secrets.PAT }}
+          pr_title: 'chore(release): pulling ${{ steps.create-release.outputs.branch_name }} into main'
+          pr_body: ":crown: *An automated PR*"
+          pr_reviewer: 'bardisg'
```
Comment (itsdebs) [inline .github/workflows/draft-new-release.yml:99]: Is it possible to add "desu" as reviewer too?

File: .github/workflows/deploy-pubdev.yml (hunk 1 above)
Comment (bardisg) [reply .github/workflows/deploy-pubdev.yml:66]: above url also contains the sdk DM integration packages

File: .github/workflows/draft-new-release.yml (hunk 2 above)
Comment (bardisg) [reply .github/workflows/draft-new-release.yml:99]: sure

**Review by itsdebs — APPROVED** [review]

**Pull request merged**